test/e2e/yaml/e2e-test-local.yaml

```diff
@@ -0,0 +1,78 @@
+---
+# core config
+varFile: ""
+toolFile: ""
+state:
+  backend: local
+  options:
+    stateFile: devstream.state
+
+---
+# variables config
+defaultBranch: main
+githubUsername: GITHUBUSERNAME
```

Comment:
It looks like the user needs to configure this variable, but how do we let the user know that it needs to be configured here?

Probably the easiest way is to add a comment here to remind the user that it must be changed.